config/config.go

```diff
@@ -975,6 +985,10 @@ func (c *Config) parseForwardedHeaders() error {
 		}
 	}
 
+	if c.ForwardedHeaders.Proto == "" && c.ForwardedHeaders.OriginalURL {
+		return errors.New("invalid forwarded header configuration X-Original-URL requires X-Forwarded-Proto")
```

Comment:
Lets use `fmt.Errorf` like elsewhere

Reply:
It is ok to use `fmt.Errorf` without arguments, see e.g. https://github.com/golang/go/issues/17173
Since both are equivalent it makes sense to stick to the one used elsewhere (`errors` package was not even imported before this chnage)

Reply:
% grep -cr errors.New * | awk -F : 'BEGIN{i=0} {i=i+$2} END{print "count: "i}'
count: 325
% grep -cr fmt.Errorf * | awk -F : 'BEGIN{i=0} {i=i+$2} END{print "count: "i}'
count: 414

Reply:
X-Original-URL was dropped so nothing to do